pkg/apis/allocation/v1/gameserverallocation.go

```diff
@@ -166,6 +172,24 @@ type ListSelector struct {
 	MaxAvailable  int64  `json:"maxAvailable"`
 }
 
+// CounterAction is an optional action that can be performed on a Counter at allocation.
+// Action: "Increment" or "Decrement" the Counter's Count (optional). Must also define the Amount.
+// Amount: The amount to increment or decrement the Count (optional). Must be a positive integer.
+// Capacity: Update the maximum capacity of the Counter to this number (optional). Min 0, Max int64.
+type CounterAction struct {
+	Action   *string `json:"action,omitempty"`
+	Amount   *int64  `json:"amount,omitempty"`
```

Comment:
Non blocking: In theory this could be `int64`, since a 0 amount is the same as none - but I also get the desire for consistency in the struct, so more of a question than an opinion 🤷🏻

Reply:
Yeah, I mostly did this for consistency.